### ISSUE TITLE
[Bug]: Persian text is not accurate

### ISSUE THREAD

**Issue by emohandesi** [Jigsaw-Code/outline-apps#2303]
### Application

Outline Client

### Describe the bug

The Persian texts of the Outline client app are not accurate. They are mostly made using auto-translations made by apps. Users will not have a correct understanding of them.

### Steps to reproduce

Change the language to Persian (فارسی) in the Outline app.
I have corrected them in `client/src/www/messages/fa.json`, but I do not have permission to push my branch into this repository.

### Outline Version

1.14.0

### What operation system are you using?

Android, iOS, Windows

**Comment by sbruens** [comment]
Feel free to share the suggested updates here (or link to your branch) and I will run them by our translators! Thanks for helping us improve the app.

**Comment by emohandesi** [comment]
Thank you so much for your cooperation. I have made the changes and tested them.
I have also committed the code into my local branch, but I do not have the permission to push into the Jigsaw repository. The following is my commit:
```
commit 4be3dd82cab00d0ad679ad2d4bc5cd565a062bcc (HEAD -> emohandesi/persian_text)
Author: Ehsan Mohandesi <emohandesi@microsoft.com>
Date:   Tue Dec 10 15:50:21 2024 -0600

    fix(client): corrected the Persian texts (#2303)
```

```
$ git branch
* emohandesi/persian_text
  master
$ git push --set-upstream origin emohandesi/persian_text
Select an authentication method for 'https://github.com/':
  1. Device code (default)
  2. Personal access token
option (enter for default): 1
To complete authentication please visit https://github.com/login/device and enter the following code:
XXXX-XXXX
remote: Permission to Jigsaw-Code/outline-apps.git denied to emohandesi.
fatal: unable to access 'https://github.com/Jigsaw-Code/outline-apps.git/': The requested URL returned error: 403
```

You can view the changes in [fa.json](https://github.com/user-attachments/files/18086279/fa.json).

**Comment by emohandesi** [comment]
I still see the old text. I was wondering what the latest update with this is. Can I raise a pull request with this fix?
